### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by snwhite** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

File: calibration/DOSTAD/CGINS-DOSTAD-00462__20201120.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+462,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_462_Multipoint_Calibration_2020-11-20.pdf"
+462,CC_csv,"[2.60585E-03, 1.10012E-04, 2.25393E-06, 1.89243E+02, -2.40919E-01, -4.65361E+01, 3.73760E+00]",
```
Comment (asmith-whoi) [inline calibration/DOSTAD/CGINS-DOSTAD-00462__20201120.csv:3]: OK

File: calibration/DOSTAD/CGINS-DOSTAD-00463__20201122.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+463,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_463_Multipoint_Calibration_2020-11-22.pdf"
+463,CC_csv,"[2.58528E-03, 1.10840E-04, 2.16400E-06, 1.62108E+02, -2.46740E-01, -3.98702E+01, 3.22007E+00]",
```
Comment (asmith-whoi) [inline calibration/DOSTAD/CGINS-DOSTAD-00463__20201122.csv:3]: OK

File: calibration/DOSTAD/CGINS-DOSTAD-00465__20201122.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+465,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_465_Multipoint_Calibration_2020-11-22.pdf"
+465,CC_csv,"[2.59425E-03, 1.09111E-04, 2.24929E-06, 2.06549E+02, -2.62493E-01, -5.08780E+01, 4.10097E+00]",
```
Comment (asmith-whoi) [inline calibration/DOSTAD/CGINS-DOSTAD-00465__20201122.csv:3]: OK

File: calibration/DOSTAD/CGINS-DOSTAD-00512__20201122.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+512,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_512_Multipoint_Calibration_2020-11-22.pdf"
+512,CC_csv,"[2.66393E-03, 1.13860E-04, 2.27847E-06, 1.87276E+02, -2.47635E-01, -5.21275E+01, 3.70112E+00]",
```
Comment (asmith-whoi) [inline calibration/DOSTAD/CGINS-DOSTAD-00512__20201122.csv:3]: OK

File: calibration/DOSTAD/CGINS-DOSTAD-00514__20201122.csv [hunk 5]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+514,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_514_Multipoint_Calibration_2020-11-22.pdf"
+514,CC_csv,"[2.53888E-03, 1.08423E-04, 2.15603E-06, 1.68142E+02, -2.51008E-01, -4.29735E+01, 3.39845E+00]",
```
Comment (asmith-whoi) [inline calibration/DOSTAD/CGINS-DOSTAD-00514__20201122.csv:3]: OK

File: calibration/PHSENE/CGINS-PHSENE-P0138__20201130.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0138,CC_ea434,17372.0, source_file PHSEN-E_SAMI-pH_SN_P0138_Validation_Certificate_2020-11-30.pdf
+P0138,CC_ea578,94.1,
+P0138,CC_eb434,2284.1,
+P0138,CC_eb578,38676.5,
+P0138,CC_ind_off,0,
+P0138,CC_ind_slp,1,
+P0138,CC_psal,35,Constant
```
Comment (asmith-whoi) [inline calibration/PHSENE/CGINS-PHSENE-P0138__20201130.csv:8]: OK

File: calibration/PHSENE/CGINS-PHSENE-P0139__20201223.csv [hunk 7]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0139,CC_ea434,17372.0, source_file PHSEN-E_SAMI-pH_SN_P0139_Validation_Certificate_2020-12-23.pdf
+P0139,CC_ea578,94.1,
+P0139,CC_eb434,2284.0,
+P0139,CC_eb578,38676.0,
+P0139,CC_ind_off,0,
+P0139,CC_ind_slp,1,
+P0139,CC_psal,35,Constant
```
Comment (asmith-whoi) [inline calibration/PHSENE/CGINS-PHSENE-P0139__20201223.csv:8]: OK

**Review by asmith-whoi — APPROVED** [review]

File: calibration/ADCPSN/CGINS-ADCPSN-21002__20201114.csv [hunk 8]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+21002,CC_scale_factor1,0.45,constant; source_file ADCPS-N_WHLS75-1500-I_SN_21002_Configuration_Summary_2020-11-14.pdf
+21002,CC_scale_factor2,0.45,constant
+21002,CC_scale_factor3,0.45,constant
+21002,CC_scale_factor4,0.45,constant
+21002,CC_bin_size,0,in cm; return from vendor - not allocated yet
+21002,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+21002,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/ADCPSN/CGINS-ADCPSN-21002__20201114.csv:8]: Looks good

File: calibration/ADCPTG/CGINS-ADCPTG-20495__20201218.csv [hunk 9]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20495,CC_scale_factor1,0.45,constant; source_file ADCPT-G_WHS150-1500-I_SN_20495_Configuration_Summary_2020-12-18.pdf
+20495,CC_scale_factor2,0.45,constant
+20495,CC_scale_factor3,0.45,constant
+20495,CC_scale_factor4,0.45,constant
+20495,CC_bin_size,0,in cm; return from vendor - not allocated yet
+20495,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+20495,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/ADCPTG/CGINS-ADCPTG-20495__20201218.csv:8]: Looks good

File: calibration/CTDBPF/CGINS-CTDBPF-50143__20201222.csv [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+16-50143,CC_pa2,1.128801e-11,
+16-50143,CC_ptca0,5.211367e+05,
+16-50143,CC_ptca1,-1.402291e+02,
+16-50143,CC_ptca2,-1.191482e+00,
+16-50143,CC_ptcb0,2.501138e+01,
+16-50143,CC_ptcb1,-5.250000e-04,
+16-50143,CC_ptcb2,0.000000e+00,
+16-50143,CC_ptempa0,1.071975e+02,
+16-50143,CC_ptempa1,-4.249363e+01,
+16-50143,CC_ptempa2,-1.223338e+00,
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/CTDBPF/CGINS-CTDBPF-50143__20201222.csv:23]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00127__20201122.csv [hunk 11]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+127,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_127_Multipoint_Calibration_2020-11-22.pdf"
+127,CC_csv,"[2.53272E-03, 1.07244E-04, 2.08744E-06, 1.24728E+02, -1.81609E-01, -3.10160E+01, 2.52627E+00]",
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/DOSTAD/CGINS-DOSTAD-00127__20201122.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00129__20201120.csv [hunk 12]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+129,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_129_Multipoint_Calibration_2020-11-20.pdf"
+129,CC_csv,"[2.58227E-03, 1.08282E-04, 2.20346E-06, 2.19033E+02, -2.81742E-01, -4.96450E+01, 4.38832E+00]",
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/DOSTAD/CGINS-DOSTAD-00129__20201120.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00427__20201122.csv [hunk 13]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+427,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_427_Multipoint_Calibration_2020-11-22.pdf"
+427,CC_csv,"[2.58444E-03, 1.08688E-04, 2.20241E-06, 2.28620E+02, -3.14344E-01, -5.54698E+01, 4.59266E+00]",
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/DOSTAD/CGINS-DOSTAD-00427__20201122.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00460__20201129.csv [hunk 14]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+460,CC_conc_coef,[0.00000E+00, 1.00000E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_460_Multipoint_Calibration_2020-11-29.pdf"
+460,CC_csv,"[2.76820E-03, 1.15058E-04, 2.40587E-06, 1.52335E+02, -2.15616E-01, -3.79270E+01, 3.05961E+00]",
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/DOSTAD/CGINS-DOSTAD-00460__20201129.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00461__20201122.csv [hunk 15]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+461,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_461_Multipoint_Calibration_2020-11-22.pdf"
+461,CC_csv,"[2.48684E-03, 1.04333E-04, 2.16480E-06, 1.92784E+02, -2.49492E-01, -5.01902E+01, 3.96993E+00]",
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/DOSTAD/CGINS-DOSTAD-00461__20201122.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00462__20201120.csv (hunk 1 above)
Comment (batryn) [reply calibration/DOSTAD/CGINS-DOSTAD-00462__20201120.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00463__20201122.csv (hunk 2 above)
Comment (batryn) [reply calibration/DOSTAD/CGINS-DOSTAD-00463__20201122.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00465__20201122.csv (hunk 3 above)
Comment (batryn) [reply calibration/DOSTAD/CGINS-DOSTAD-00465__20201122.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00512__20201122.csv (hunk 4 above)
Comment (batryn) [reply calibration/DOSTAD/CGINS-DOSTAD-00512__20201122.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00514__20201122.csv (hunk 5 above)
Comment (batryn) [reply calibration/DOSTAD/CGINS-DOSTAD-00514__20201122.csv:3]: Looks good

File: calibration/PHSENE/CGINS-PHSENE-P0138__20201130.csv (hunk 6 above)
Comment (batryn) [reply calibration/PHSENE/CGINS-PHSENE-P0138__20201130.csv:8]: Looks good

File: calibration/PHSENE/CGINS-PHSENE-P0139__20201223.csv (hunk 7 above)
Comment (batryn) [reply calibration/PHSENE/CGINS-PHSENE-P0139__20201223.csv:8]: Looks good

**Review by batryn — APPROVED** [review]
Looks good

**Pull request merged**